demos/demo_calibration.cpp

```diff
@@ -59,7 +56,7 @@ int main(int, char**)
 {
     enable_ctrl_c()
```

Comment:
missing a semi-colon here, this is why the format looks weird

Reply:
Good catch.  Yes, it does not seem to be compiled.  Given this and the fact that it is using the old `calibrate` function which is marked deprecated, should I fix the file or better remove it?

Reply:
I deleted it now.  Let me know in case you want to keep it, then I'll undo this commit.